cocos/asset/assets/image-asset.jsb.ts

```diff
@@ -169,6 +171,9 @@ imageAssetProto._syncDataToNative = function () {
     }
     else {
         this.setData(this._nativeData._data);
+        if (this._nativeData.mipmapLevelDataSize) {
+            this._setMipmapLevelDataSize(this._nativeData.mipmapLevelDataSize);
```

Comment:
Is there an `_setMipmapLevelDataSize` exported to JS on native platform? I only see a method called `setMipmapLevelDataSize` without `_` prefix.